### DISCUSSION TITLE
"New live session detected" notification

### DISCUSSION THREAD

**Discussion by AtlAntA118** [f1multiviewer/issue-tracker#38] [Feature requests]
Racecontrol had an option to give an audio notification "New Live Session Detected" . Would be a nice addition for this app as well.

With RC it worked only when the app was open which makes sense ofcourse. I had RC on autostart cause i hated missing F3 and F2 races.

**Comment by meph0** [comment]
Would like this as well. Or maybe don't poll for new live sessions, but just send out an (optional) push notification 15-5 minutes before the start of a session according to the official schedule. Of course my calendar could do that, and it usually does, but I get distracted easily and forget.
